### PR TITLE
feat(recording): Limit notification

### PR DESCRIPTION
**Pull request by hristoterezov**

### CODE REVIEW

File: lang/main.json [hunk 1]
```diff
@@ -395,6 +395,8 @@
         "videoQuality": "Manage call quality"
     },
     "liveStreaming": {
+        "limitNotificationDescriptionWeb": "Due to high demand your recording will be limited to {{limit}} min. For unlimited recordings try <a href={{url}} rel='noopener noreferrer' target='_blank'>{{app}}</a>.",
```
Comment (bgrozev) [inline lang/main.json:398]: Should this say "streaming" instead of "recording"?

File: react/features/recording/actions.web.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+export * from './actions.any';
+
+/**
+ * Signals that a started recording notification should be shown on the
+ * screen for a given period.
+ *
+ * @param {string} streamType - The type of the stream ({@code file} or
+ * {@code stream}).
+ * @param {string} participantName - The participant name that started the recording.
```
Comment (bgrozev) [inline react/features/recording/actions.web.js:18]: There's no `participantName` parameter, as far as I see (am I missing something?).

File: react/features/recording/components/web/RecordingLimitNotificationDescription.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import { connect } from '../../../base/redux';
+
+/**
+ * The type of the React {@code Component} props of {@link RecordingLimitNotificationDescription}.
+ */
+type Props = {
+    /**
+     * The limit of time for the recording.
+     */
+    _limit: number,
```
Comment (bgrozev) [inline react/features/recording/components/web/RecordingLimitNotificationDescription.js:15]: Units?

File: react/features/recording/components/web/RecordingLimitNotificationDescription.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+    /**
+     * The name of the payed app with unlimited recordings.
```
Comment (bgrozev) [inline react/features/recording/components/web/RecordingLimitNotificationDescription.js:18]: Is it important that the application requires paiment? I suggest to use either just `appName` and `appUrl`, or `unlimitedAppName` and `unlimitedAppUrl`. If you prefer the reference to payment, it should be spelled `paid`.

File: lang/main.json (hunk 1 above)
Comment (hristoterezov) [reply lang/main.json]: Fixed!

File: react/features/recording/components/web/RecordingLimitNotificationDescription.js (hunk 4 above)
Comment (hristoterezov) [reply react/features/recording/components/web/RecordingLimitNotificationDescription.js:19]: Agree! Will remove the pay stuff!

File: react/features/recording/actions.web.js (hunk 2 above)
Comment (hristoterezov) [reply react/features/recording/actions.web.js:18]: it was copy/paste error

Comment (hristoterezov) [reply react/features/recording/actions.web.js]: Fixed!

File: react/features/recording/components/web/RecordingLimitNotificationDescription.js (hunk 3 above)
Comment (hristoterezov) [reply react/features/recording/components/web/RecordingLimitNotificationDescription.js:16]: Added `minutes` to the JSDoc.

File: react/features/recording/components/web/RecordingLimitNotificationDescription.js (hunk 4 above)
Comment (hristoterezov) [reply react/features/recording/components/web/RecordingLimitNotificationDescription.js]: Fixed!

**Comment by hristoterezov** [comment]
@bgrozev I think everything is addressed! Could you please approve so that I can merge it?

**Comment by bgrozev** [comment]
I only looked at it very superficially, I don't understand the code wel enough to feel comfortable approving.

**Comment by hristoterezov** [comment]
@bgrozev Thanks for the comments, tho!

File: react/features/recording/actions.web.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ * {@code stream}).
+ * @returns {showNotification}
+ */
+export function showRecordingLimitNotification(streamType: string) {
+    const isLiveStreaming = streamType === JitsiMeetJS.constants.recording.mode.STREAM;
+
+    return showNotification({
+        description: <RecordingLimitNotificationDescription isLiveStreaming = { isLiveStreaming } />,
+        titleKey: isLiveStreaming ? 'dialog.liveStreaming' : 'dialog.recording'
+    }, 100000);
```
Comment (damencho) [inline react/features/recording/actions.web.js:26]: Is this value intentionally different than the one for mobile?

Comment (hristoterezov) [reply react/features/recording/actions.web.js:26]: Not really :) there's one additional zero. Will remove it.

Comment (hristoterezov) [reply react/features/recording/actions.web.js]: Fixed!

File: config.js [hunk 6]
```diff
@@ -210,6 +210,21 @@ var config = {
...
+    //    // The recording limit in minutes. Note: This number appears in the notification text
+    //    // but doesn't enforce the actual recording time limit. This should be configured in
+    //    // jibri!
+    //    limit: 60,
+    //
+    //    // The name of the app with unlimited recordings.
+    //    appName: 'Unlimited recordings APP',
+    //
+    //    // The URL of the app with unlimited recordings.
+    //    appURL: 'https://unlimited.recordings.app.com/'
```
Comment (surendra10125) [inline config.js:225]: what is thiis line mean ?

**Pull request merged**